graphene-writer/src/main/kotlin/com/graphene/writer/store/GrapheneStoreConfig.kt

```diff
@@ -1,25 +1,20 @@
 package com.graphene.writer.store
 
-import com.graphene.writer.input.graphite.property.InputGraphiteCarbonProperty
-import com.graphene.writer.store.data.CassandraDataStoreHandler
-import com.graphene.writer.store.data.CassandraDataStoreHandlerProperty
-import com.graphene.writer.store.data.CassandraFactory
-import org.springframework.context.annotation.Bean
 import org.springframework.context.annotation.Configuration
 
 @Configuration
 class GrapheneStoreConfig {
 
-  @Bean
-  fun cassandraDataStoreHandler(
-    inputGraphiteCarbonProperty: InputGraphiteCarbonProperty,
-    handlerProperty: CassandraDataStoreHandlerProperty
-  ): StoreHandler {
-
-    return CassandraDataStoreHandler(
-      inputGraphiteCarbonProperty,
-      handlerProperty,
-      CassandraFactory()
-    )
-  }
+//  @Bean
+//  fun cassandraDataStoreHandler(
+//    inputGraphiteCarbonProperty: InputGraphiteCarbonProperty,
+//    handlerProperty: CassandraDataStoreHandlerProperty
+//  ): StoreHandler {
+//
+//    return SimpleDataStoreHandler(
+//      inputGraphiteCarbonProperty,
+//      handlerProperty,
+//      CassandraFactory()
+//    )
+//  }
```

Comment:
How to load data store handler if I set up to OffsetBased or else?

Reply:
The store handler is loaded from DataFetchHandlerConfig.kt in the way that is similar to StoreHandlerFactory.kt in Writer. Refactored many classes to have similar concept to Writer for better readability.
This class won't be used anymore, so I will delete the class.

Reply:
I got it :) 
Could you delete this file and resolve conflicts for the merge?

Reply:
Done :)